documentation/Working_on_Bot/README.md

```diff
@@ -0,0 +1,257 @@
+# Working on the Bot
+
+- [Creating Bot and Attaching it to a Development Server](https://github.com/CSSS/wall_e/wiki/2.-Creating-Bot-and-Attaching-it-to-a-Development-Server)  
+- [Running the Bot](#running-the-bot)  
+  - [With the Database](#with-the-database)
+    - [Step 1. Re-creating the docker base image](#step-1-re-creating-the-docker-base-image)
+    - [Step 2. Launching the Bot](#step-2-launching-the-bot)
+  - [Without the Database](#without-the-database)
+    - [Step 1. Re-creating the docker base image](#step-1-re-creating-the-docker-base-image-1)
+    - [Step 2. Launching the Bot](#step-2-launching-the-bot-1)
+- [Testing the Bot](#testing-the-bot)
+  - [Step 1. Run through the linter](#step-1-run-through-the-linter)
+  - [Step 2. Testing on the CSSS Bot Test Server](#step-2-testing-on-csss-bot-test-server)
+- [Making a PR to master](https://github.com/CSSS/wall_e/wiki/3.-Making-a-PR-to-master)  
+- [Test Cases](#test-cases)  
+- [Reporting Issues](https://github.com/CSSS/wall_e/wiki/4.-Reporting-Issues)  
+- [FAQs](#faqs)  
+
+
+## Running the Bot
+>If you encounter any errors doing the following commands, feel free to add it to the [FAQs section](documentation/Working_on_Bot#faqs) for future reference :)
+
+>Due to some compatibility issues that occured in the past when wall_e was dockerized only on the server, the dev environment for wall_e was changed so that each developer doesn't have to fight with OS issues that come with some python modules that are a bit iffy. as long as you can get `docker` and `docker-compose` working, you are golden to work on wall_e.
+
+Pre-requisites: `git`, [`docker`](https://docs.docker.com/install/linux/docker-ce/debian/#set-up-the-repository) and [`docker-compose`](https://docs.docker.com/compose/install/#install-compose)
+
+1. Fork the [Wall-e Repo](https://github.com/CSSS/wall_e.git)  
+2. clone the repo
+3. Wall_E Setting Specification.
+   1. Wall_e needs some settings in order to determine how some parts should be treated. The biggest component this impacts is whether or not you use wall_e with or without a dockerized database. [refer to the wiki page on the ini file](https://github.com/CSSS/wall_e/wiki/5.-contents-of-local.ini) for all the settings that wall_e reads from when doing local dev work.
```

Comment:
>Wall_e needs some settings in order to determine how some parts should be treated.

This is extremely vague.

>The biggest component this impacts is whether or not you use wall_e with or without a dockerized database.

"whether or not" and  "with or without" is redundant.

>[refer

[**R**efer

I'm going to stop commenting on missing capital letters at the beginning of sentences.

Reply:
fixed with 8ea8e94bdd7503f01b839a3f260f57ddcaa99a59

Reply:
>Wall_e needs some settings in order to determine which commands should be enabled.
Still confusing. How about "needs parameter specifications"?

Reply:
fixed with bf77e0412e2126fea80d396803cde50cd7e20787